scriptworker/cot/verify.py

```diff
@@ -1879,6 +1896,10 @@ def verify_repo_matches_url(repo, url):
         bool: ``True`` if the repo matches the url.
 
     """
+    # Translate git@github.com:... to a path:
+    repo = repo.replace('git@github.com:', 'ssh://github.com/', 1)
+    if repo.endswith('.git'):
+        repo = repo[:-4]
```

Comment:
We could create a helper function to do these things for us (tweak `get_source_repo` ?) but I'm ok with it like this.